library/pod-security-policy/privileged-containers/template.yaml

```diff
@@ -16,14 +16,27 @@ spec:
 
         violation[{"msg": msg, "details": {}}] {
             c := input_containers[_]
-            c.securityContext.privileged
+            input_allow_privileged_container(c)
             msg := sprintf("Privileged container is not allowed: %v, securityContext: %v", [c.name, c.securityContext])
         }
 
+        input_allow_privileged_container(c) {
+            not has_field(c, "securityContext")
```

Comment:
if `securityContext` is not provided, k8s would default it to `false` which should not trigger this violation.

Reply:
yeah, I have just checked this and you are right! thank you so much @ritazh